playwright/_impl/_browser_context.py

```diff
@@ -599,8 +599,12 @@ async def _inner_close() -> None:
         await self._channel.send("close", {"reason": reason})
         await self._closed_future
 
-    async def storage_state(self, path: Union[str, Path] = None) -> StorageState:
-        result = await self._channel.send_return_as_dict("storageState")
+    async def storage_state(
+        self, path: Union[str, Path] = None, indexedDB: bool = None
```

Comment:
why not `indexed_db`?

Reply:
Our generator layer has some magical logic which maps `indexed_db` (user consuming API) to `indexedDB` (driver). The reason for this is that we like to use the `protocol.yaml` compatible property names in the implementation so we can simply pass `locals()` to the driver.